.github/pull_request_template.md

```diff
@@ -13,34 +14,35 @@ It may include
 
 The summary may omit some of these details if this PR fixes a single ticket that includes these details. It is the reviewer's discretion. 
 
-## [Optional] Changes
-Multiple changes are not recommended, so this section should normally be omitted. Unforunately, they are sometimes unavoidable. If there are multiple logical changes, list them separately.
+## Changes [Optional] 
+Multiple changes are not recommended, so this section should normally be omitted. Unfortunately, they are sometimes unavoidable. If there are multiple logical changes, list them separately.
 
 1. `'foo'` is replaced by `'bar'`
 2. `'fizzbuzz'` is optimized for gas
 
-# [Optional] How Has This Been Tested?
+## How Has This Been Tested? [Optional] 
 
 Did you need to run manual tests to verify this change? If so, please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
 
-# [Optional] :warn: Does this require multiple approvals?
+## :warning: Does this require multiple approvals? [Optional]
 Please explain which reason, if any, why this requires more than one approval.
 - [ ] Is it security related?
 - [ ] Is it a significant process change?
 - [ ] Is it a significant change to architectural, design?
 
-# Checklist:
+---
+## Checklist:
 
-## Code quality
+### Code quality
 - [ ] I have written clear commit messages
 - [ ] I have performed a self-review of my own code
-- [ ] I have scoped this change as narrowly as possible
+- [ ] This change does not have an unduly wide scope
 - [ ] I have separated logic changes from refactor changes (formatting, renames, etc.)
-- [ ] I have commented my code wherever necessary
-- [ ] I have added tests that prove my fix is effective or that my feature works
-## Project management
+- [ ] I have commented my code wherever necessary (can be 0)
+- [ ] I have added tests that prove my fix is effective or that my feature works, if necessary
+### Project management
 - [ ] I have applied the [appropriate labels](https://www.notion.so/Team-working-agreements-2a95c926bb5642e5a5c42e4b74a9dd24#b304e56734a74dfbb341b8b4b27b1c0c)
-- [ ] I have linked to relevant issues
-- [ ] I have added dependent tickets
+- [ ] I have linked to all relevant issues (can be 0)
+- [ ] I have added all dependent tickets (can be 0)
 - [ ] I have assigned myself to this PR
-- [ ] I have chosen the appropriate pipeline
+- [ ] I have associated this PR to a GitHub Project
```

Comment:
[required]: I don't understand this. We aren't using a Github Project at the moment. What's your intended meaning here? Can you explain the action with a screenshot?

"pipeline" refers to the dropdown in the top right:

<img width="1248" alt="Screen Shot 2021-01-26 at 9 04 06 AM" src="https://user-images.githubusercontent.com/8432675/105870384-74e7ce00-5fb5-11eb-9a66-8dadc96b1c26.png">